### DISCUSSION TITLE
Required RAM for rotor in hover (high fidelity)

### DISCUSSION THREAD

**Discussion by NM1293** [byuflowlab/FLOWUnsteady#168] [Q&A]
Hello Everyone,

I am trying to run the rotor in hover tutorial, in high fidelity: https://flow.byu.edu/FLOWUnsteady/examples/rotorhover-aero/

However, during the execution, Julia killed itself. If I understand well, this could depend on a Run Out Of Memory problem.

I am trying to run the case on a 16 Gb RAM computer.

Has anyone been able to launch the case without Run Out of Memory issues? If so, what are the specifications of the PC used? How much RAM did you need?

Thank you.

**Comment by EdoAlvarezR** [comment]
You can either reduce `nrevs` to make the simulation shorter (less particles), or you can manually overwrite the maximum number of particles `max_particles` to whatever your resources can handle (the simulation will likely break at some point when it hits the maximum number of particles though, so I would recommend changing `nrevs` instead, or you could just reduce the fidelity (less particles) ).

**Reply by NM1293** [reply]
Dear Dr @EdoAlvarezR ,
thank you for these suggestions.

I have already run the simulation with mid-low and mid-high fidelity, and the calculations have been completed successfully. Now, I am attempting to run the high fidelity case to compare results with the available data, ensuring the correct implementation of the code (this could also be done with a reduced number of revolutions).

However, having an idea of the required computational power would be very useful for me, as I will likely run other more complex cases in the future.

Thank you.
